packages/vscode-extension/src/commands/CompareFile.ts

```diff
@@ -33,17 +33,12 @@ export default class CompareFile extends FileImportCommon {
 		if (projectInfoService.isSuiteAppProject()) {
 			if (!this.activeFileIsUnderSuiteAppsAppIdFolder(projectInfoService)) {
 				return this.unsuccessfulValidation(
-					this.translationService.getMessage(
-						COMPARE_FILE.ERROR.SUITEAPP_NOT_ALLOWED_FOLDER,
-						projectInfoService.getApplicationId()
-					)
+					this.translationService.getMessage(COMPARE_FILE.ERROR.SUITEAPP_NOT_ALLOWED_FOLDER, projectInfoService.getApplicationId())
```

Comment:
No action is needed for this comment. Changes in this file are only about formatting, if possible please avoid committing files with no logic changes in the code.